packages/myst-to-md/src/misc.ts

```diff
@@ -0,0 +1,56 @@
+import type { Handle, Info, State } from 'mdast-util-to-markdown';
+import { fileWarn } from 'myst-common';
+import type { VFile } from 'vfile';
+import type { Parent, Validator } from './types';
+
+function comment(node: any): string {
+  return `% ${node.value}`;
```

Comment:
Do we do any new line handling here?

Reply:
I think I was maybe referring to `node.value = 'something with \n newelines'`, I don't think this can happen easily, so probably not important at the moment!